lib/routemap.h

```diff
@@ -167,7 +167,7 @@ struct route_map {
 	bool deleted;         /* If 1, then this node will be deleted */
 
 	/* How many times have we applied this route-map */
-	uint64_t applied;
```

Comment:
I think you should not remove applied( and replace it with applied_counter), these are 2 separate things being tracked here.  The applied is a metric of how many times this route-map has been used.  This is different than keeping track of anyone using this route-map.

As a note I can have a route-map that was used at one point in time so the applied would have some number but has been removed from active configuration usage and it is no-longer being used.

Let's keep both.

Reply:
Hi @donaldsharp ,

I faced some issues while undoing changes to specific commit . So, I have pulled back all my commits and addressed your suggested changes and raised a new pull request. 

Please find the following new pull req with updated changes.
https://github.com/FRRouting/frr/pull/3518

Regards,
Rajesh.